transport_nantes/press/views.py

```diff
@@ -296,3 +297,30 @@ def get_image_extension(file_contents):
         press_mention.og_image_url.save(image_filename, files.File(image_fp))
         press_mention.save()
         return None
+
+
+@csrf_protect
+@permission_required('press.press-editor')
+def check_for_duplicate(request, *args, **kwargs):
+    """Check if the provided URL already exists in PressMention."""
+    url = request.GET.get("url")
```

Comment:
It wont throw. 

`request.GET` is a dict
dict.get() is defined like so : https://docs.python.org/3/library/stdtypes.html#dict.get

there is a `default` kwarg that is set to `None`
So get, if it doesn't find the key, returns None. That's the difference with `myDict["key that doesn't exist"]` that will throw if the key doesn't exist. 

From python doc : 
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).